src/components/ec-modal/ec-modal.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <div
+    v-if="showModal"
+    class="ec-modal"
+  >
+    <div
+      class="ec-modal__content"
+      :class="{'ec-modal--lg': large}"
+    >
+      <header class="ec-modal__header">
+        <slot name="header" />
+        <ec-icon
+          v-if="showCloseIcon"
+          class="ec-modal__close"
+          name="simple-close"
+          :size="24"
+          @click="closeModal()"
+        />
+      </header>
+
+      <main class="ec-modal__main">
+        <slot name="main" />
+      </main>
+
+      <footer class="ec-modal__footer">
+        <div
+          v-if="showHelpLink"
+          v-ec-tooltip="tooltipConfig"
+          class="ec-modal__help-link"
+        >
+          <ec-icon
+            class="ec-mr--8"
+            name="simple-help"
+            :size="18"
+          />
+          Need help?
```

Comment:
This text must be coming from props or slot. We do not have translations in chameleon-components, so all texts must be configurable from EBO.